lib/src/progress_bar.dart

```diff
@@ -77,7 +78,7 @@ class _VideoProgressBarState extends State<VideoProgressBar> {
       ),
     );
 
-    return chewieController.draggableProgressBar
+    return widget.draggableProgressBar
```

Comment:
Is there a reason you introduce a new parameter instead of passing the user defined value into the chewieController so that the value is accessable through all of the plugin?

Reply:
Seconded. Everything should revisit l revolve around the `ChewieController`.

Reply:
The intension is to use `MaterialVideoProgressBar` widget without depending on the `ChewieControllerProvider`. Hence, other video package like video_player, video_editor can be used for displaying the video and`MaterialVideoProgressBar` for custom controls.

Below is the sample code of using `MaterialVideoProgressBar` with other package.
```dart
//with video_editor
Stack(
      fit: StackFit.expand,
      children: [
        CropGridViewer.preview(
          controller: widget.controller,
        ),
        MaterialVideoProgressBar(
          widget.controller.video,
        ),
      ],
    ),

//with video_player
 Stack(
      fit: StackFit.expand,
      children: [
        VideoPlayer(widget.videoController),
        MaterialVideoProgressBar(
          widget.videoController,
        ),
      ],
    ),
```
I do notice `draggableProgressBar` property is added but no value is being passed, hence, I have added some changes in .